playground/frontend/playground_components/lib/src/build_metadata.g.dart

```diff
@@ -0,0 +1,9 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
+//
+// This file is generated during deployment to contain data about the commit.
+// The copy of this file stored in the repository is for development purpose
+// so the project can be started locally.
+// It is safe to commit changes here everytime you run code generation.
+const buildCommitHash = '0d41f157812fdc3227a6ea93429d3096aa03a850' ;
```

Comment:
I don't like that we commit these values into the repo. Can we write something like `unknown` here?

Reply:
`+`